src/pseudopeople/interface.py

```diff
@@ -193,27 +196,35 @@ def generate_american_community_survey(
     :param config: An optional override to the default configuration. Can be a path
         to a configuration YAML file or a dictionary.
     :param year: The survey date year (format YYYY) to include in the dataset. Will
-        return an empty pd.DataFrame if there are no data with this year. If None is
+        raise a ValueError if there is no data for this year. If None is
         provided, data from all years are included in the dataset.
     :param state: The state string to include in the dataset. Either full name or
-        abbreviation (e.g., "Ohio" or "OH"). Will return an empty pd.DataFrame if there are no
-        data pertaining to this state. If None is provided, data from all locations are
+        abbreviation (e.g., "Ohio" or "OH"). Will raise a ValueError if there is no
+        data for this state. If None is provided, data from all locations are
         included in the dataset.
     :param verbose: Log with verbosity if True.
     :return: A pd.DataFrame of simulated ACS data.
     :raises ConfigurationError: An incorrect config is provided.
     :raises DataSourceError: An incorrect pseudopeople input data source is provided.
     """
     user_filters = []
-    if year:
-        user_filters.extend(
+    if year is not None:
+        user_filters.append(
             [
-                (DATASETS.acs.date_column_name, ">=", pd.Timestamp(f"{year}-01-01")),
-                (DATASETS.acs.date_column_name, "<=", pd.Timestamp(f"{year}-12-31")),
+                (
```

Comment:
I have changed how we are declaring these pd.Timestamps because it was not parsing year correctly.

Reply:
There's no difference between the `append` version you've added here and the `extend` version we had before.

Reply:
Agreed with Zeb. I think it's cleaner to extend a single list rather than appending single filters twice.